src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook_test.go

```diff
@@ -549,6 +551,67 @@ func TestValidateUpdate_NoError(t *testing.T) {
 		err := c.ValidateUpdate(redpandaCluster)
 		assert.Error(t, err)
 	})
+
+	decreaseCases := []struct {
+		initial    string
+		target     string
+		error      bool
+		lowerBound string
+	}{
+		{
+			initial: "2",
+			target:  "2500m",
+			error:   false,
+		},
+		{
+			initial: "2",
+			target:  "1001m",
+			error:   false,
```

Comment:
Isn't `2` > `1001m`?
Am I missing something?

Reply:
Yeah, considering how CPU request are currently mapped to cores, "1001m" becomes 2 cores (`--smp=2`), so these kinds of reductions are allowed..

Reply:
👍